customtkinter/windows/widgets/ctk_button.py

```diff
@@ -488,7 +488,8 @@ def _on_enter(self, event=None):
 
             # set text_label bg color to button hover color
             if self._text_label is not None:
-                self._text_label.configure(bg=self._apply_appearance_mode(inner_parts_color))
+                self._text_label.configure(bg=self._apply_appearance_mode(inner_parts_color),
+                                           fg=self._apply_appearance_mode(self._fg_color))
```

Comment:
> fg=self._apply_appearance_mode(self._fg_color)

the `fg=` means the text color?

and if yes does this mean that on hover the text color will become the buttons color?